app/src/main/java/com/farmerbb/taskbar/content/TaskbarPosition.java

```diff
@@ -0,0 +1,198 @@
+/* Copyright 2020 Braden Farmer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.farmerbb.taskbar.content;
+
+import android.view.Surface;
+
+public class TaskbarPosition {
+    // TODO Change string to enum to get more security checking.
```

Comment:
One thing to keep in mind is that these constants are also defined inside a string array inside the XML resources.  See https://github.com/farmerbb/Taskbar/blob/master/app/src/main/res/values/donottranslate.xml#L54-L63

If these are eventually going to be converted to enums then we will need to implement some kind of string-to-enum conversion.  The "Position on screen" preference maps directly to that array inside resources, which makes the resources version the canonical one.

Reply:
My first thought is to change them to enum type, and the build system will help us to check the usage of task position to ensure all used values are listed by enum. But it will need some work, so I just add a `TODO` to it for reminding. Current version maybe is enough, so I will remove `TODO` from code.